### DISCUSSION TITLE
Pagination control for datagrid in winui3

### DISCUSSION THREAD

**Discussion by ApoorvaPetkar** [microsoft/microsoft-ui-xaml#7736] [Q&A]
Hi,

can someone please suggest a component for pagination purpose? Not able to find any in the winui3 desktop app and in the community toolkit I just found PipPager- https://docs.microsoft.com/en-us/windows/apps/design/controls/pipspager.

Looking for something similar
![image](https://user-images.githubusercontent.com/34087324/190560903-c7860fbe-cc87-42e5-92b7-4acea846d1ee.png)

Is there any other component I can look into? Thanks in advance.

**Comment by marcelwgn** [comment]
For WinUI 3, beside the PipsPager control I cannot think of any control that you could use for pagination purposes. In WinUI 2 there was the PagerControl (#60) which does exactly what you want, however I am not sure why it is not supported in WinUI 3. @marissamatt or @gabbybilka do you happen to know why that is the case?

**Comment by AndrewKeepCoding** [comment]
I suggested a [UserControl](https://stackoverflow.com/a/73741739/2411960) to this very question in StackOverflow.

**Reply by marcelwgn** [reply]
Alternatively, you could also try and see the PagerControl implementation from Uno Platform if you want to implement it yourself.
